draft-ietf-quic-transport.md

```diff
@@ -1590,7 +1590,7 @@ frames. A receiver MAY send timestamps for non-retransmittable packets.
 A receiver MUST not send timestamps in unprotected packets.
 
 A sender MAY intentionally skip packet numbers to introduce entropy into the
-connection, to avoid opportunistic acknowledgement attacks.  The sender MUST
+connection, to avoid opportunistic acknowledgement attacks.  The sender SHOULD
```

Comment:
Generally I worry with a SHOULD no-one will end up implementing it because people will realize that it requires maintaining a lot of state. I like the spirit of Ian's wording but it's super confusing. SHOULD seems like the right thing here. I think if we acknowledge why it is a SHOULD it might convince more people to implement it along with a suggestion of how to implement it. 

For example a sender could implement this by keeping track of all the packets sent and by advancing a smallest ackable packet periodically.